bentoml/artifact/fastai_model_artifact.py

```diff
@@ -46,8 +46,10 @@ def __init__(self, name):
             raise SystemError("fast ai requires python 3.6 version or higher")
 
         if fastai is None:
-            raise ImportError("fastai package is required to use "
-                              "bentoml.artifacts.FastaiModelArtifact")
+            raise ImportError(
+                "fastai package is required to use "
+                "bentoml.artifacts.FastaiModelArtifact"
+            )
```

Comment:
Strange I didn't change this. Maybe my IDE auto changed this?

Reply:
did you run the ./script/format.sh script?

Reply:
oh yeah, it is that script.